Mathlib/Analysis/LocallyConvex/Bounded.lean

```diff
@@ -251,13 +251,13 @@ theorem TotallyBounded.isVonNBounded {s : Set E} (hs : TotallyBounded s) :
   simp_rw [← nhds_prod_eq, id.def] at h'
   rcases h.basis_left h' U hU with ⟨x, hx, h''⟩
   rcases hs x.snd hx.2.1 with ⟨t, ht, hs⟩
-  refine' Absorbs.mono_right _ hs
-  rw [ht.absorbs_iUnion]
+  refine Absorbs.mono_right ?_ hs
+  rw [ht.absorbs_biUnion]
   have hx_fstsnd : x.fst + x.snd ⊆ U := add_subset_iff.mpr fun z1 hz1 z2 hz2 ↦
     h'' <| mk_mem_prod hz1 hz2
   refine' fun y _ => Absorbs.mono_left _ hx_fstsnd
   rw [← Set.singleton_vadd, vadd_eq_add]
-  exact (absorbent_nhds_zero hx.1.1).absorbs.add hx.2.2.absorbs_self
+  exact Absorbs.add (absorbent_nhds_zero hx.1.1 _) hx.2.2.absorbs_self
```

Comment:
Is dot notation not working here?

Reply:
It timeouts for a reason I fail to understand.

Reply:
Can you leave a comment explaining that?

Reply:
Done (I also added `Absorbent.vadd_absorbs`, so it's not visible here).